$NetBSD: patch-ab,v 1.3 2011/05/28 16:32:10 bad Exp $

- Try to correct some name of files.
- disassemble.rb relies on pre-requisites that do not work on 1.8.7

--- ../ruby-debug-0.10.4.gemspec.orig	2011-05-28 15:27:41.000000000 +0000
+++ ../ruby-debug-0.10.4.gemspec	2011-05-28 15:28:55.000000000 +0000
@@ -67,23 +67,22 @@
 - VERSION
 - Rakefile
 - cli/ruby-debug.rb
-- cli/ruby-debug/processor.RB
+- cli/ruby-debug/processor.rb
 - cli/ruby-debug/helper.rb
 - cli/ruby-debug/debugger.rb
 - cli/ruby-debug/commands/source.rb
 - cli/ruby-debug/commands/set.rb
-- cli/ruby-debug/commands/raise.RB
+- cli/ruby-debug/commands/raise.rb
 - cli/ruby-debug/commands/breakpoints.rb
 - cli/ruby-debug/commands/list.rb
 - cli/ruby-debug/commands/display.rb
-- cli/ruby-debug/commands/continue.RB
+- cli/ruby-debug/commands/continue.rb
 - cli/ruby-debug/commands/trace.rb
 - cli/ruby-debug/commands/frame.rb
 - cli/ruby-debug/commands/show.rb
 - cli/ruby-debug/commands/save.rb
 - cli/ruby-debug/commands/variables.rb
 - cli/ruby-debug/commands/threads.rb
-- cli/ruby-debug/commands/disassemble.RB
 - cli/ruby-debug/commands/irb.rb
 - cli/ruby-debug/commands/tmate.rb
 - cli/ruby-debug/commands/kill.rb
@@ -99,7 +98,7 @@
 - cli/ruby-debug/commands/eval.rb
 - cli/ruby-debug/commands/info.rb
 - cli/ruby-debug/commands/continue.rb
-- cli/ruby-debug/commands/source.RB
+- cli/ruby-debug/commands/source.rb
 - cli/ruby-debug/commands/control.rb
 - cli/ruby-debug/commands/condition.rb
 - cli/ruby-debug/interface.rb
